src/Client.php

```
@@ -450,6 +451,10 @@ protected function createPipeline(array $options = null, $callable = null)
             $class = 'Predis\Pipeline\Pipeline';
         }
 
+        if( $this->connection instanceof RelayConnection) {
+            $class = 'Predis\Pipeline\Relay';
+        }
+
```

Relay does not support `fire-and-forget`, we need to throw an exception here.